src/pages/PreviewScreen.jsx

```diff
@@ -4,31 +4,51 @@ import { Preview } from "@100mslive/hms-video-react";
 import { AppContext } from "../store/AppContext";
 import getToken from "../services/tokenService";
 
-const PreviewScreen = () => {
+const PreviewScreen = ({ getUserToken }) => {
```

Comment:
why do we have to pass getUserToken as prop? it is a util function. it can be imported and used here itself